rust/agama-lib/src/proxies.rs

```diff
@@ -1,18 +1,6 @@
-//! # DBus interface proxies for: `org.opensuse.Agama*.**.*`
+//! D-Bus interface proxies for: `org.opensuse.Agama*.**.*`
 //!
 //! This code was generated by `zbus-xmlgen` `3.1.0` from DBus introspection data.`.
-//!
-//!
-//! These DBus objects implements
-//! [standard DBus interfaces](https://dbus.freedesktop.org/doc/dbus-specification.html),
-//! (`org.freedesktop.DBus.*`) for which the following zbus proxies can be used:
-//!
-//! * [`zbus::fdo::PropertiesProxy`]
-//! * [`zbus::fdo::IntrospectableProxy`]
-//!
-//! …consequently `zbus-xmlgen` did not generate code for the above interfaces.
-//! Also some proxies can be used against multiple services when they share interface.
```

Comment:
what is reason to remove these notes? I actually find it quite useful when I look for some proxies.

Reply:
Well, it is exactly the same text copied over and over. I can bring it back anyway.